eden/scm/edenscm/ext/github/submit.py

```diff
@@ -288,6 +288,7 @@ async def rewrite_pull_request_body(
         pr_numbers_and_num_commits,
         index,
         repository,
+        reviewstack=(ui.configbool("pr", "body.reviewstack"))
```

Comment:
I renamed the config to `github.pull-request-include-reviewstack`. The config section is usually the extension name. In this case it is "github".